### DISCUSSION TITLE
Next.js Commerce and Spree integration

### DISCUSSION THREAD

**Discussion by jamessparksolutions** [spree/spree#11421] [Show and tell]
![next-and-spree-1-1024x577](https://user-images.githubusercontent.com/80058347/138449889-eb8c29a1-e04f-42fe-8b9e-1a1b815be7ad.png)

[Next.js Commerce](https://nextjs.org/commerce) is an all-in-one React starter kit for high-performance e-commerce sites, built by [Vercel](https://vercel.com/home). With a few clicks, Next.js developers can clone, deploy and fully customize their own store. Front-end development has never been easier – developers love working with it.

Create delightful online experiences and make data-driven iterative improvements independent from the Spree backend. This way, you can be truly agile with your frontend work while at the same time keeping that Spree upgrade path clean and future-proof.

![spree_commerce_spree_njc_2@2x](https://user-images.githubusercontent.com/80058347/138444466-6005c65a-d3fe-43d6-9511-4838fdc7721f.png)

### Next.js Commerce and Spree demo
Below you can check out the Next.js Commerce default storefront designs. Feel free to explore the Next.js and Spree demo here: https://spree.vercel.app/.

**HOME PAGE**
![Screenshot 2021-09-16 at 10 15 55](https://user-images.githubusercontent.com/80058347/138444717-b4f20e7f-43c7-44e9-93c7-3b999efa35d0.png)

**PRODUCT LISTING PAGE**
![Screenshot 2021-09-16 at 10 16 26](https://user-images.githubusercontent.com/80058347/138444766-a928ab62-d077-4866-a54b-4b2c3517a158.png)

**PRODUCT DETAIL PAGE**
![Screenshot 2021-09-16 at 10 16 05](https://user-images.githubusercontent.com/80058347/138444797-e033b3e5-7351-4731-ba74-cc630c140361.png)

### How to install Next.js Commerce
For an update-to-date guide on how to integrate Next.js Commerce with Spree, please refer to our developer documentation: https://dev-docs.spreecommerce.org/storefronts/next.js-commerce

For more information on the Next.js Commerce and Spree integration, please refer to: https://github.com/vercel/commerce

**Comment by iswarpatel123** [comment]
https://dev-docs.spreecommerce.org/storefronts/next.js-commerce

this link is dead. is there a guideline/starter for nextjs v+ spree?
